recipes-core/varstored/varstored_git.bb

```diff
@@ -0,0 +1,76 @@
+SUMMARY = "Package for managing guest EFI variables"
+LICENSE = "BSD"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=8444b396c3cde7d8fe18ae36a3638a55"
+
+inherit useradd xc-rpcgen-c
+
+DEPENDS = " \
+    dbus \
+    dbus-glib \
+    libseccomp \
+    libxml2 \
+    libxcdbus \
+    openssl \
+    xen-tools \
+"
+
+CFLAGS_prepend += "-I${STAGING_INCDIR}/dbus-1.0 -I${STAGING_LIBDIR}/dbus-1.0/include -I${STAGING_INCDIR}/glib-2.0 -I${STAGING_LIBDIR}/glib-2.0/include "
+LDFLAGS_prepend += " -levent -lgobject-2.0 -lglib-2.0 -ldbus-glib-1 -lxcdbus "
+
+# lock to this SRCREV until we uprev Xen to >4.12
+SRCREV = "f8c2656be1cfca35bb9297b97d9af962f51f19f9"
+PV = "0+git${SRCPV}"
+
+S = "${WORKDIR}/git"
+
+FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
+
+SRC_URI = " \
+    git://github.com/xapi-project/varstored.git;protocol=https;branch=master \
+    file://0001-add-oxtdb-varstore.patch \
+"
+
+# download the ever-growing community dbx.auth file, which contains
+# a list of known malicious guids that we should never boot with.
+SRC_URI += " \
+    https://uefi.org/sites/default/files/resources/dbxupdate_x64.bin;downloadfilename=dbx.auth \
+"
+
+SRC_URI[sha256sum] = "46ba1f2a0a2ed7aabe20f9b7b2a8d717cb0b514cea83c7a1a24fe25f6b208784"
+
+USERADD_PACKAGES = "${PN}"
+USERADD_PARAM_${PN} = "--system --no-create-home \
+                       --shell /bin/false \
+                       --groups varstored \
+                       --gid 415 \
+                       --uid 416 \
+                       varstored"
+GROUPADD_PARAM_${PN} = "--system --gid 415 varstored"
```

Comment:
I was going to comment on not having hardcoded uid & gid.  Looking ahead, varstored only takes numeric values, and varstored-watch hardcodes these numbers.  Which is a little unfortunate, but understandable.

Reply:
I'm not too big a fan of this either. We were having issues with the IDs in OE and hardcoding seemed to solve it. I noticed that other recipes have the IDs hardcoded as well (trousers maybe?), so I stuck with it.